gradio/blocks.py

```diff
@@ -878,6 +882,43 @@ def expects_oauth(self):
             for block in self.blocks.values()
         )
 
+    def unload(self, fn: Callable):
```

Comment:
For this event, agree that there is no need for outputs. But in theory, `inputs` could be useful. E.g. if you want to log the final value of a component as the user leaves the page. My suggestion would to have all of the regular parameters but simply warn if a user tries to put in outputs that it will unused.

Reply:
The tricky thing here is that when this is triggered, the client closed the connection so we can't get data from the client anymore

Reply:
Ah right, makes sense nvm then not needed.